sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -837,7 +837,8 @@ class SQLMetricsSuite extends SharedSparkSession with SQLMetricsTestUtils
     }
   }
 
-  test("SPARK-34399: Add job commit duration metrics for DataWritingCommand") {
+  test("SPARK-34399: Add job commit duration metrics for DataWritingCommand" +
+    " and SPARK-48397: Data write time metric for DataWritingCommand") {
```

Comment:
Piggyback on an existing test case to check the newly added metric of the metrics map from `org.apache.spark.sql.execution.datasources.BasicWriteJobStatsTracker#metrics`.